kotlinpoet/src/jvmTest/kotlin/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -702,6 +702,30 @@ class FunSpecTest {
     }.hasMessageThat().isEqualTo("Using both context receivers and context parameters is not allowed")
   }
 
+  @Test fun contextParameterInAddStatement() {
+    val loggerType = ClassName("java.util.logging", "Logger")
+    val configType = ClassName("com.example", "Config")
+
+    val logger = ContextParameter("logger", loggerType)
+    val config = ContextParameter("config", configType)
+
+    val processData = FunSpec.builder("processData")
+      .contextParameter("logger", loggerType)
+      .contextParameter("config", configType)
```

Comment:
same:

```suggestion
      .contextParameter(logger)
      .contextParameter(config)
```

Reply:
Nice addition!

 I double check current implementation, removed overrides that I think actually not required (I was basically copy-pasting receivers implementation initially, and now done more assessment to what actually have to be done).
 And added context parameters into `toBuilder` functions.

Reply:
Looks great, thank you!